bundler.d/assets.rb

```diff
@@ -1,21 +1,18 @@
 group :assets do
   gem 'ace-rails-ap', '~> 4.0.0'
-  gem 'sass-rails', '~> 3.2'
+  gem 'sass-rails', '~> 5.0'
```

Comment:
Would 4.x be enough?  Our SCL package is 4.0.3, so it'd be useful to be able to reuse it.